benchmarks/portable/chaos.py

```
@@ -72,17 +73,27 @@ def wait_pg_ready(timeout: int = 30):
             time.sleep(0.5)
     raise RuntimeError("Postgres not ready")
 
+def compose_env(pg_image: str) -> dict[str, str]:
+    return {**os.environ, "POSTGRES_IMAGE": pg_image}
+
 def start_postgres():
+    pg_image = os.environ.get("POSTGRES_IMAGE", DEFAULT_PG_IMAGE)
     subprocess.run(
         ["docker", "compose", "up", "-d", "--wait"],
```

Ruff (0.15.9)

[error] Starting a process with a partial executable path

(S607)